ResearchKit/Consent/ORKVisualConsentStepViewController.m

```diff
@@ -506,10 +512,24 @@ - (void)observedScrollViewDidScroll:(UIScrollView *)scrollView {
 }
 
 - (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated {
+    [self showViewController:viewController forward:forward animated:animated preloadNextViewController:YES];
+}
+
+- (void)showViewController:(ORKConsentSceneViewController *)viewController forward:(BOOL)forward animated:(BOOL)animated preloadNextViewController:(BOOL)preloadNextViewController {
     [self showViewController:viewController
                      forward:forward
                     animated:animated
-                  completion:nil];
+                  completion:^(BOOL finished) {
+                      if (preloadNextViewController) {
+                          ORKConsentSceneViewController *nextViewController = [self viewControllerForIndex:[self currentIndex]+1];
```

Comment:
ORKConsentSceneViewController has this code when setting the section:

```
    UIImage *image = nil;
    if (consentSection.type == ORKConsentSectionTypeCustom) {
        image = consentSection.customImage;
    } else {
        image = ORKImageForConsentSectionType(consentSection.type);
    }
```

Perhaps we could put this in a private method on ORKConsentSection.